### ISSUE TITLE
ID of requestable__pickup appears more than once

### ISSUE THREAD

**Issue by axamei** [pulibrary/requests#347]
Violation: ID attributes must be unique. `id="requestable__pickup"` can only appear once per page.

**Comment by sdellis** [comment]
This one is a doozy. Instead of "hiding" and "showing" the different select boxes via `display:none;` (which all need the same id) we will need to completely remove or add the element. WIP...

**Comment by kevinreiss** [comment]
@sdellis couldn't we just append the unique "item" id available to each "requestable" row rendered by the form to make these values unique?
